src/components/Sidebar/index.js

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Modal from 'react-modal';
 
-import Box from '../Box';
+import Button from 'components/Button';
+import Box from 'components/Box';
+import NewPost from 'components/NewPost';
 
-import './style.scss';
+import style from './style.scss';
```

Comment:
I believe that if you do it this way, the style is not just added automatically to this page. So 'sidebar' or 'sidebar__actions' classnames' css won't be here

Reply:
I think the styles are getting loaded, the issue with not using the `style` is that I need it in order to send it to the `Modal`